pod-security/restricted/require-run-as-nonroot/require-run-as-nonroot.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     policies.kyverno.io/category: Pod Security Standards (Restricted)
     policies.kyverno.io/severity: medium
     policies.kyverno.io/subject: Pod
-    kyverno.io/kubernetes-version: "1.22-1.23"
+    kyverno.io/kubernetes-version: "1.22-1.22"
```

Comment:
I think we should just have a min kubernetes version annotation for this. Not sure what `1.22-1.22` means? Is it only for 1.22?